2017-12-11-openstack-manageiq-issues.md

```diff
@@ -0,0 +1,25 @@
+
+
+---
+Title: Introduction to OpenStack and ManageIQ
```

Comment:
Title should be same as written in file name

Reply:
It is not introductory post. So change it to appropriate. Use `title` instead of Title

Reply:
yeah, I have modified the title.